demos/human_pose_estimation_demo/python/human_pose_estimation_demo.py

```diff
@@ -97,6 +97,8 @@ def build_argparser():
 def get_model(ie, args, aspect_ratio):
     if args.architecture_type == 'ae':
         Model = models.HpeAssociativeEmbedding
+    elif args.architecture_type == 'hrnet':
+        Model = models.HpeHRNet
```

Comment:
Now I using one class for `ae` and `hrnet` models, but I using new type to set special parameters of model class.

Reply:
currently demo work with this model for both architecture_type `ae` and `higherhrnet` without visible difference what looks a bit strange. Why do we need to distinguish them?

Reply:
These types use model with different parameters. We need to distinguish them so that user don't need to think about parameters `delta` and `padding_mode`.

Reply:
just as I said before, demo work with this model when architecture_type is ae. How it is possible then? And if it is possible somehow, then why should one care about those delta and paddings?

Reply:
intel HPE models and higher-hrnet model use similar postprocessing to get result. This postprocessing implemented in `HpeAssociativeEmbedding` model, but there are parameters that affect to pre- and postprocessing algorithm. So in order not to create a separate model class, we added the type `higherhrnet`.